hcloud/network.go

```diff
@@ -226,6 +236,9 @@ type NetworkCreateOpts struct {
 	Subnets []NetworkSubnet
 	Routes  []NetworkRoute
 	Labels  map[string]string
+	// ExposeRoutesToVSwitch indicates if the routes from this network should be exposed to the vSwitch connection.
+	// The exposing only takes effect if a vSwitch connection is active.
+	ExposeRoutesToVSwitch bool
```

Comment:
Shouldn't this field be optional when creating a network ? And have nil instead of false ?

Reply:
Its optional, if you don't specify it, go will default this to `false`, which is also the default from the API.

I think not having a pointer here is more ergonomic for the user, otherwise they would need to pass `ExposeRoutesToVSwitch: hcloud.Ptr(true)`